### DISCUSSION TITLE
OAuth access for FastHub

### DISCUSSION THREAD

**Discussion by TysonMN** [hedgehogqa/fsharp-hedgehog#376] [General]
@moodmosaic, did you see in the #Hedgehog cannel of Slack that I requested OAuth access for FastHub within in the hedgehogaq organization on GitHub?

**Comment by moodmosaic** [comment]
Sorry if I missed that. I am currently traveling. Happy to look into that tonight though. You'd want this for the F# repos or for the whole org? (I've never used it before so I don't know much about it.)

**Reply by moodmosaic** [reply]
(Jumping on a plane soon btw.)

**Reply by moodmosaic** [reply]
@TysonMN it should be all set now. Sorry for the delay. (On my way back from a [month of travelling](https://www.instagram.com/p/CTX8kettFl8/).)
